engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -1229,7 +2135,7 @@ private String getSimpleName(Attribute<?> attribute) {
   }
 
   private Map<String,String> getApplicationProperties(List<AppConf> appConfs){
-    Map<String, String> appProps = new HashMap<String, String>();
+    Map<String, String> appProps = Maps.newHashMap();
```

Comment:
There is a bias towards guava factory methods throughout the code in Malhar, and some people are militant about doing it. So I've gotten into the habit of doing it. It's also easier to read without the generics for some people.